src/v3/test/integration/identify-with-password.test.tsx

```diff
@@ -37,10 +37,10 @@ describe('identify-with-password', () => {
     });
 
     await findByText('Sign in', { selector: 'button' });
-    const usernameEl = await findByTestId('identifier-hint') as HTMLLabelElement;
+    const usernameEl = await findByLabelText(/Username/) as HTMLInputElement;
```

Comment:
We have to use `/Username/` because optional Odyssey fields will bring in the "Optional" string as text content when trying to select a field by its label